deploy/helm/charts/values.yaml

```diff
@@ -132,6 +132,10 @@ lvmPlugin:
     containerRuntime: containerd
     readIopsPerGB: ""
     writeIopsPerGB: ""
+  # The TCP network address where the prometheus metrics endpoint will listen.
+  # The default is empty string, which means metrics endpoint is disabled.
+  # If not set, service will not be created to expose this endpoint to serviceMonitor and listen-address flag will not be set.
+  metricsListenAddress: 9500
```

Comment:
above comment & variable name suggests that it is net address, but you are using it as port number. Either update the var name & comment as metricsPort or use it a string along with default as ":9500".